src/main/java/com/book/controller/CartItemsController.java

```diff
@@ -0,0 +1,32 @@
+package com.book.controller;
+
+import com.book.model.CartItemsEntity;
+import com.book.model.UsersEntity;
+import com.book.service.CartItemsService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@CrossOrigin(origins = "http://localhost:3000")
+@RestController
+@RequestMapping("/cart")
+@RequiredArgsConstructor
+public class CartItemsController {
+    private final CartItemsService cartItemsService;
+
+    @PostMapping("/getCartItems")
+    public ResponseEntity<List<CartItemsEntity>> getCartItems(@RequestBody UsersEntity usersEntity) {
```

Comment:
if you need only the user id in this method, then use only `id` as `@RequestParam` and change the method to GET.
GET method is suitable for retrieving the data (according to REST principle)